devfile.yaml

```diff
@@ -11,30 +11,31 @@ schemaVersion: 2.1.0
 metadata:
   name: che-code
 components:
+
   - name: dev
     container:
-      image: quay.io/che-incubator/che-code-dev:insiders
+      image: quay.io/che-incubator-pull-requests/che-code-dev:pr-246-dev-amd64
       memoryLimit: 12Gi
+      memoryRequest: 512Mi
+      cpuRequest: 500m
       cpuLimit: 3500m
-      endpoints:
-        - exposure: public
-          name: dev
-          secure: true
-          protocol: http
-          targetPort: 8000
```

Comment:
With this endpoint, we get a handy `dev` item to quickly open a running dev instance of Che-Code:

![image](https://github.com/che-incubator/che-code/assets/1636395/6cedac35-7002-4f9b-b593-0c531c19e397)

Without this endpoint declared, one needs to use a notification to add a redirect manually:

![image](https://github.com/che-incubator/che-code/assets/1636395/48f6212c-2f44-4251-adb7-49ff11575ae2)

as there's no endpoint, by default:

![image](https://github.com/che-incubator/che-code/assets/1636395/95f57824-6478-4649-b64d-20ae7aece4d6)

Why do we need to remove it?

Reply:
This is it! 
I think why I am constantly asked to expose the port each time when I launch secondary instance of VS Code..